Gemfile.lock

```diff
@@ -1,62 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.1.1)
+    better_errors (2.5.0)
       coderay (>= 1.0.0)
-      erubis (>= 2.6.6)
+      erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (9.0.6)
-    coderay (1.1.1)
-    commander (4.4.3)
-      highline (~> 1.7.2)
-    database_cleaner (1.6.1)
+    byebug (10.0.2)
+    coderay (1.1.2)
+    commander (4.4.7)
+      highline (~> 2.0.0)
+    database_cleaner (1.7.0)
     diff-lcs (1.3)
-    erubis (2.7.0)
-    event_sourcery (0.13.0)
-    event_sourcery-postgres (0.3.0)
-      event_sourcery (>= 0.10.0)
+    erubi (1.7.1)
+    event_sourcery (0.22.0)
+    event_sourcery-postgres (0.8.0)
+      event_sourcery (>= 0.14.0)
       pg
-      sequel (~> 4.38)
-    highline (1.7.8)
-    method_source (0.8.2)
-    mustermann (1.0.0)
+      sequel (>= 4.38)
+    highline (2.0.0)
+    method_source (0.9.2)
+    mustermann (1.0.3)
     pg (0.20.0)
-    pry (0.10.4)
+    pry (0.12.2)
```

Comment:
### [pry](http://pryrepl.org)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 0.10.4 → 0.12.2

[_~~change-log~~, [source-code](http://github.com/pry/pry)_]





<details>
<summary>Commits</summary>

A change of **590** commits. See the full changes on [the compare page](https://github.com/pry/pry/compare/v0.10.4...v0.12.2).

These are the first 10 commits:
- (e0da73e) [update changelog.md](https://github.com/pry/pry/commit/e0da73e73dcb8cce2437eb87b3c51757834fc7a8)
- (eeda531) [use Integer instead of `1.class`](https://github.com/pry/pry/commit/eeda5313f87558ccde83396f0c975eda3127b513)
- (8d030c3) [Merge pull request #1611 from pry/respond_to-2.4-warnings](https://github.com/pry/pry/commit/8d030c3faae899534128c527b3053797cfec0c8d)
- (505cccf) [rename Pry::Config::Lazy to Pry::Config::Memoization (ref #1…](https://github.com/pry/pry/commit/505cccf69193f979ec67cd2bebf4ff9f409cf0d6)
- (ba49f4c) [Merge branch 'master' of github.com:pry/pry](https://github.com/pry/pry/commit/ba49f4caa6998b427e246d105e5b91d8bc01ac03)
- (72a17c5) [last_default might not be a Pry::Config::Memoized](https://github.com/pry/pry/commit/72a17c5b246a1e2ac1871ded978979c370c4fa55)
- (d46e0d2) [eager load all Pry::Config::Memoized objects that may be in …](https://github.com/pry/pry/commit/d46e0d207f96df711cda764248eadf428220013e)
- (030c617) [fix eager_load! while](https://github.com/pry/pry/commit/030c61715792c915fc14c4621e1e9e3660f9b728)
- (3743228) [add docs [ci skip]](https://github.com/pry/pry/commit/3743228846cda5cf3c90bda7ceb787ef71cfbaad)
- (beaad80) [whereami?!?!?!?! fix #1597](https://github.com/pry/pry/commit/beaad80d0c1c1bf81ee464cdf9ae9a9e9b38c1df)


</details>